packages/cdk/lib/roles/policies/nextflow-batch-policy.ts

```diff
@@ -0,0 +1,28 @@
+import * as iam from "monocdk/aws-iam";
+
+export interface NextflowBatchPolicyProps {
+  account: string;
+  region: string;
+}
+
+export class NextflowBatchPolicy extends iam.PolicyDocument {
+  constructor(props: NextflowBatchPolicyProps) {
+    const nextflowJobArn = `arn:aws:batch:${props.region}:${props.account}:job-definition/nf-ubuntu:*`;
+
+    super({
+      assignSids: true,
+      statements: [
+        new iam.PolicyStatement({
+          effect: iam.Effect.ALLOW,
+          actions: ["batch:DescribeJobDefinitions", "batch:ListJobs", "batch:DescribeJobs", "batch:DescribeJobQueues", "batch:DescribeComputeEnvironments"],
+          resources: ["*"],
+        }),
+        new iam.PolicyStatement({
+          effect: iam.Effect.ALLOW,
+          actions: ["batch:RegisterJobDefinition"],
```

Comment:
I think(?) Nextflow will clean up old job definitions when it's done, so we may need DeregisterJobDefinition or whatever as well.

Reply:
From what I've seen, both nextflow and cromwell both don't clean up the job definitions